prime-router/src/main/kotlin/azure/DatabaseAccess.kt

```diff
@@ -1352,6 +1355,106 @@ class DatabaseAccess(private val create: DSLContext) : Logging {
             .fetchInto(Action::class.java)
     }
 
+    /**
+     * Returns all permissions associated with an organization by name
+     * @param organizationName an exact organization name
+     * @param txn an optional database transaction
+     */
+    fun fetchPermissionsByOrgName(
+        organizationName: String,
+        txn: DataAccessTransaction? = null
+    ): List<Permission> {
+        val ctx = if (txn != null) DSL.using(txn) else create
+        return ctx.select(PERMISSION.asterisk())
+            .from(PERMISSION)
+            .join(PERMISSION_ORGANIZATION)
+            .on(PERMISSION.PERMISSION_ID.eq(PERMISSION_ORGANIZATION.PERMISSION_ID))
+            .where(
+                PERMISSION_ORGANIZATION.ORGANIZATION_NAME.eq(organizationName)
+                    .and(PERMISSION.ENABLED.eq(true))
+            )
+            .limit(100)
```

Comment:
Would it make sense to make the limit an optional parameter instead?  I realize 100 might be a biggggg upper bound we don't expect to reach through API calls, but it might make more sense to pass back all Permission records for the given Organization (especially since that's stated in the comment above :P) and manually limit ourselves with a new parameter instead.  What do you think about this?

Reply:
Sure, sounds good.

Reply:
To be honest we probably don't even need a limit. We only have 4 flags currently and they aren't suppose to stick around long term.